bom/pom.xml

```diff
@@ -52,6 +52,7 @@
     <vertx.version>3.4.1</vertx.version>
     <vertx-proton.version>3.4.1</vertx-proton.version>
     <vertx-mqtt.version>3.4.1</vertx-mqtt.version>
+    <vertx-metrics.version>3.4.1</vertx-metrics.version>
```

Comment:
I do not think that we need a separate version for this since we are using the standard metrics that come with vertx itself, don't we?